core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -2401,6 +2401,13 @@ object ZIO extends ZIOCompanionPlatformSpecific {
   def collectParN[R, E, A, B](n: Int)(in: Iterable[A])(f: A => ZIO[R, Option[E], B]): ZIO[R, E, List[B]] =
     foreachParN(n)(in)(a => f(a).optional).map(_.flatten)
 
+  /**
+   * Similar to Either.cond, evaluate the predicate,
+   * return the given A as success if predicate returns true, and the given E as error otherwise
+   */
+  def cond[R, E, A](predicate: Boolean, result: => A, error: => E): ZIO[R, E, A] =
```

Comment:
```suggestion
  def cond[E, A](predicate: Boolean, result: => A, error: => E): IO[E, A] =
```

Reply:
True, the 'R' is totally useless in my version